packages/world/package.json

```diff
@@ -31,5 +32,11 @@
       "path": "./node_modules/cz-conventional-changelog"
     },
     "script_path": "../../scripts/package"
+  },
+  "devDependencies": {
```

Comment:
devDependencies for any/all packages, aside from @mineral-ui dependencies, should be installed only at the root level.

As a result of the current setup, there are some errors in the console when running the bootstrap command.  You can see them in the travis console or when running locally.  I have verified that removing the devDependencies from the World package resolves this issue.